sdk/tables/azure-data-tables/README.md

```diff
@@ -1,74 +1,78 @@
 
-Azure Data Tables is a NoSQL data storing service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
-Tables scale as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
 The Azure Data Tables client can be used to access Azure Storage or Cosmos accounts.
 
 
 # Usage
 * Storing structured data in the form of tables	# Usage
 * Quickly querying data using a clustered index
 
-[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk) | [Package (PyPI)](https://pypi.org) | [API reference documentation](https://aka.ms/azsdk/python/tables/docs) | [Product documentation](https://docs.microsoft.com/azure/storage/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk)
-For code examples, see [MyService Management](https://docs.microsoft.com/python/api/overview/azure/)
-on docs.microsoft.com.
+[Source code][source_code] | [Package (PyPI)][Tables_pypi] | [API reference documentation][Tables_ref_docs] | [Product documentation][Tables_product_doc] | [Samples][Tables_samples]
 
 ## Getting started
+The Azure Data Tables can be accessed using an Azure Storage or a CosmosDB account.
 
 ### Prerequisites
 * Python 2.7, or 3.5 or later is required to use this package.
-* You must have an [Azure subscription](https://azure.microsoft.com/free/) and an
-[Azure storage account](https://docs.microsoft.com/azure/storage/common/storage-account-overview) to use this package
- or you must have a [Azure Cosmos Account](https://docs.microsoft.com/azure/cosmos-db/account-overview).
+* You must have an [Azure subscription][azure_subscription] and an
+[Azure storage account][azure_storage_account] to use this package
+ or an [Azure Cosmos Account][azure_cosmos_account].
 
 ### Install the package
-Install the Azure Data Tables client library for Python with [pip](https://pypi.org/project/pip/):
+Install the Azure Data Tables client library for Python with [pip][pip_link]:
 
 ```bash
 pip install --pre azure-data-tables
 ```
 
-### Create a storage account
-If you wish to create a new cosmos storage account, you can use the [Azure Cosmos DB](https://docs.microsoft.com/azure/cosmos-db/create-cosmosdb-resources-portal)
-If you wish to create a new storage account, you can use the
-[Azure Portal](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal),
-[Azure PowerShell](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell),
-or [Azure CLI](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli):
+#### Create a storage account
```

Comment:
I'd move this and the next section under the prerequisites. You mention that you need to create them there so this is coming a bit late. In prerequisites I'd write something like: 
* To use this package you must have an [Azure subscription][azure_subscription] and either:
    1. [Azure storage account][azure_storage_account] 
 or 
    1. [Azure Cosmos Account][azure_cosmos_account].

And then add numbering to the sections for creating storage or cosmos